src/kubernetes_service.py

```diff
@@ -6,7 +6,8 @@
 
 from typing import List, Tuple
 
-import kubernetes
```

Comment:
[clarification] why is this change necessary (or why is it part of this pr?)

Reply:
It's necessary because the **kubernetes** pypi package doesn't use typing much, and the linter struggles to find class properties if you don't specify it like that !